db/dcrpg/upgrades.go

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) 2018, The dcrdata developers.
```

Comment:
Now just `// Copyright (c) 2018, The Decred developers` since the project was absorbed into Decred.